### DISCUSSION TITLE
Materia Messages Still Blocked

### DISCUSSION THREAD

**Discussion by Illixy** [NadyaNayme/TidyChat#83] [Q&A]
While trying to meld my gatherer/crafter gear, I've noticed that some materia messages appear and some don't despite having all the options checked.  Turned on Debug, melded some random gear, and you can see that the attaches aren't showing.  The first three are single melds and the bottom three are bulk melds.

![image](https://github.com/user-attachments/assets/4a16ec78-ea94-4371-a726-384bc0ed3890)

At first I just assumed it was the bulk melds not showing but that doesn't seem to be the case.  Am I just missing something or is this an issue with the filters?

**Comment by NadyaNayme** [comment]
It looks like "attach (a|an)" depending on name settings can become "attaches" and so it no longer matches the filter.

I hate name settings. I'm not sure if I change `attach` to `(attach|attaches)` if your name is properly being normalized to "you" (Tidy Chat in an attempt to save my sanity swaps out your player name/initials whenever it finds them with "you"). 

I've updated the RegEx and will try and push an update soonish after seeing if there are any other quick fixes I can throw in the update.

The regex for this was `^you successfully attach (a|an) .+ materia [I|V|X|L|C|D|M]{1,10} to the"` and is now `^you successfully (attach|attaches) (a|an) .+ materia [I|V|X|L|C|D|M]{1,10} to the"` - again with `you` being your name normalized by Tidy Chat (assuming that is working properly - Tidy Chat won't debug print the message like that but internally that is how it reads it.)
